dll/win32/shell32/brsfolder.cpp

```diff
@@ -1168,34 +1145,24 @@ LPITEMIDLIST WINAPI SHBrowseForFolderA (LPBROWSEINFOA lpbi)
 
 /*************************************************************************
  * SHBrowseForFolderW [SHELL32.@]
- *
- * NOTES
- *  crashes when passed a null pointer
  */
-LPITEMIDLIST WINAPI SHBrowseForFolderW (LPBROWSEINFOW lpbi)
+EXTERN_C
+LPITEMIDLIST WINAPI SHBrowseForFolderW(LPBROWSEINFOW lpbi)
 {
-    browse_info info;
-    INT_PTR r;
-    HRESULT hr;
-    WORD wDlgId;
+    TRACE("%p\n", lpbi);
```

Comment:
I'm not convinced of the necessity of systematically adding such traces to every function.

Reply:
Deleted.